Lean thinking (#2)
lapaevrv/translation: principles/lean-thinking.ru.md

  —<a href="https://ru.wikipedia.org/wiki/Джеки_Мейсон" title="Джеки Мейсон - американский стендап-комик, актёр телевидения и озвучивания">Джеки Мейсон (Jackie Mason)</a>
</p></div>

> Бережливое мышление — это проверенная система, масштабируемая до уровня крупного производства, что подтверждается корпорацией Toyota и другими. Несмотря на то, что этот термин чаще применим к промышленному производству, он также используется в сфере обслуживания — как внутри Toyota так и в таких областях, здравоохранение.

Reviewer: ```suggestion
Бережливое мышление — это проверенная система, масштабируемая до уровня крупного производства, что подтверждается корпорацией Toyota и другими. Несмотря на то, что этот термин чаще применим к промышленному производству, он также используется в сфере обслуживания — как внутри Toyota так и в таких областях, как здравоохранение.
```

> Метафора (от Дона Рейнертсена (Don Reinertsen), сторонника бережливого подхода), которую мы используем, чтобы донести ключевую ошибку в понимании (и вместе с ней основную идею) — это эстафетный бег.

Reviewer: ```suggestion
Метафора (от Дона Рейнертсена [Don Reinertsen], сторонника бережливого подхода), которую мы используем, чтобы донести ключевую ошибку в понимании (и вместе с ней основную идею) — это эстафетный бег.
```
Предлагаю оставить как есть, круглые скобки в круглых скобках как-то не очень.

<figure>
  <img class="rounded shadowed" src="/img/lean-thinking/baton.jpg" alt="Watch the baton, not the runners.">
  <figcaption>Следи за эстафетной палочкой, а не за бегунами.</figcaption>
</figure>

Вообразите себе эстафету. Бегуны *топчутся на месте* в ожидании эстафетной палочки от членов своей команды. Бухгалтер финансового департамента был бы в ужасе от таких кошмарных 'потерь' из-за недостаточной загрузки и вероятно потребовал бы установить политику "95% утилизация ресурсов", чтобы удостовериться, что все бегуны загружены и "продуктивны". Наверняка он бы посоветовал, чтобы бегуны могли бы бежать три эстафеты в одно и то же время, чтобы увеличить "утилизацию ресурсов", или они могли бы бежать в гору в процессе ожидания эстафетной палочки.

Забавно... Но такое мышление лежит в основе многих традиционных процессов управления и промышленного производства. В противоположность этому, вот как звучит ключевая идея бережливого мышления:

> 

Reviewer: ```suggestion
```
Случайно или есть какой-то смысл? Я предпочитаю пробелы и переводы строк как в оригинале оставлять.

### Предыстория

> **Бережливый подход или Бережливое мышление** (**lean thinking, lean**) — это английское название, данное [инженером Toyota](http://www.leanblog.org/2010/08/who-coined-the-term-lean-and-where-is-he-today/), выполняющим дипломную работу в Массачусетском технологическом институте (MIT), системе известной сейчас как Дао Toyota (Toyota Way). Toyota — сильная, стабильная компания, которая, как можно видеть, совершенствуется на протяжении длительного времени. Так, например,  в 2014 году [журнал Forbes](http://www.forbes.com/pictures/eimi45eilh/no-1-toyota-motor/) оценил Toyota, как крупнейшую и самую дорогую автомобильную компанию в мире. Одна из глав книги "*Экстремальная Toyota*" посвящена сравнению устойчивых показателей Toyota с другими компаниями в своей отрасли.

Reviewer: ```suggestion
**Бережливый подход или Бережливое мышление** (**lean thinking, lean**) — это английское название, данное [инженером Toyota](http://www.leanblog.org/2010/08/who-coined-the-term-lean-and-where-is-he-today/), выполняющим дипломную работу в Массачусетском технологическом институте (MIT), системе известной сейчас как Путь Toyota (Toyota Way). Toyota — сильная, стабильная компания, которая, как можно видеть, совершенствуется на протяжении длительного времени. Так, например,  в 2014 году [журнал Forbes](http://www.forbes.com/pictures/eimi45eilh/no-1-toyota-motor/) оценил Toyota, как крупнейшую и самую дорогую автомобильную компанию в мире. Одна из глав книги "*Экстремальная Toyota*" посвящена сравнению устойчивых показателей Toyota с другими компаниями в своей отрасли.
```
Как я понял, существует  Toyota Way - в самой Toyota, а также была написана по его мотивам одноимённая книга, название которой на русский перевели как "Дао Toyota". В данном же случае идёт речь не о книге, поэтому я предпочитаю более точный перевод.

Author: Согласен, я тоже уже потом понял по ходу текста

Toyota далека от совершенства, и в мире существует масса идей из других подходов, о которых следует знать и которые нельзя найти в бережливом мышлении. Мы не настаиваем на том, что Toyota или бережливое мышление — это единственные модели, из которых следует извлекать уроки или просто следовать их примеру. Тем не менее, это эволюционирующая на протяжении длительного времени, эффективная система, созданная довольно надёжной и устойчивой компанией. 

> ### Столпы Бережливого Подхода - Это *Не* Инструменты или Уменьшение Потерь

Reviewer: ```suggestion
###Основы Бережливого Подхода - Это *Не* Инструменты или Уменьшение Потерь
```
Здесь я предлагаю не буквальный перевод "Pillars".

Есть несколько заблуждений относительно бережливого подхода. Этот раздел начинается с того, чтобы развеять их. Что является сутью и силой бережливого мышления и Toyota?

> > Когда я впервые начал изучать Производственную Систему Toyota (TPS), я был очарован силой [потока единичных изделий, канбаном и другими инструментами бережливого подхода]. Но на протяжении всего пути, опытные лидеры Toyota не переставали объяснять мне, что эти инструменты и техники не были ключевыми. Сила, лежащая в основе TPS, это скорее *стремление менеджмента компании постоянно инвестировать в своих людей и проповедовать культуру непрерывного улучшения*. После учёбы в течение почти 20-ти лет…..я наконец-то полностью это осознал. (курсив добавлен) [[Liker04]](http://www.amazon.com/Toyota-Way-Management-Principles-Manufacturer-ebook/dp/B000SEGIVS/ref=sr_1_1?ie=UTF8&qid=1414352313&sr=8-1&keywords=the+toyota+way)

Reviewer: ```suggestion
> Когда я впервые начал изучать Производственную Систему Toyota (TPS), я был очарован силой [потока единичных изделий, канбаном и другими инструментами бережливого подхода]. Но на протяжении всего пути, опытные лидеры Toyota не переставали объяснять мне, что эти инструменты и техники не были ключевыми. Сила, лежащая в основе TPS, это скорее *стремление менеджмента компании постоянно инвестировать в своих людей и проповедовать культуру непрерывного улучшения*. После обучения в течение почти 20-ти лет…..я наконец-то полностью это осознал. (курсив добавлен) [[Liker04]](http://www.amazon.com/Toyota-Way-Management-Principles-Manufacturer-ebook/dp/B000SEGIVS/ref=sr_1_1?ie=UTF8&qid=1414352313&sr=8-1&keywords=the+toyota+way)
```
Твой вариант лучше, но небольшая мелочь :)

Вакамацу (Wakamatsu) и Кондо (Kondo), эксперты Toyota, выражают это кратко:

> > Суть [системы Toyota] в том, чтобы каждому отдельному сотруднику была предоставлена возможность самостоятельно находить и решать проблемы, а также вносить необходимые улучшения.

Reviewer: ```suggestion
> Суть [системы Toyota] в том, чтобы каждому отдельному сотруднику была предоставлена возможность самостоятельно, своим собственным способом находить и решать проблемы, а также вносить необходимые улучшения.
```
"Своим собственным способом" на мой взгляд точнее передаёт суть "in his own way", поэтому предлагаю оставить.

> ### Инструменты Менеджмента *Не являются* Столпами Бережливого подхода

Reviewer: ```suggestion
### Инструменты Управления *Не являются* Основой Бережливого подхода
```
Если есть устойчивый термин в Lean "Инструменты Менеджмента", то согласен. Иначе предлагаю более русское слово использовать :) И всё-таки Основы а не Столпы (тоже более по-русски).

> Приведённые выше цитаты подчёркивают жизненно важный момент, потому что на протяжении многих лет существовали некие якобы «бережливые» сторонники, которые опустили бережливое мышление до механического, поверхностного уровня инструментов управления, таких как канбан и управление очередями. Эти вторичные, производные описания игнорируют основное послание экспертов Toyota, которые подчёркивают, что основная суть успешного бережливого мышления заключается в том, чтобы «сначала инвестировать в людей, а потом в продукты» в культуре «бросающей вызов статус-кво» путём непрерывного совершенствования.

Reviewer: ```suggestion
Приведённые выше цитаты подчёркивают жизненно важный момент, потому что на протяжении многих лет существовали некие якобы «бережливые» сторонники, которые опустили бережливое мышление до механического, поверхностного уровня инструментов управления, таких как канбан и управление очередями. Эти вторичные, производные описания игнорируют основное послание экспертов Toyota, которые подчёркивают, что основная суть успешного бережливого мышления заключается в том, чтобы «сначала инвестировать в людей, а потом в продукты» и в культуре «бросающей вызов статус-кво» путём непрерывного совершенствования.
```
По-моему всё-таки 'И' в культуре.

> … это случилось только после того, как Американские автопроизводители исчерпали все возможные объяснения успеха Toyota (заниженный курс иены, послушная рабочая сила, японская культура, превосходная автоматизация), тогда они наконец-то смогли признать, что ***настоящее преимущество Toyota в способности задействовать интеллектуальный потенциал ‘обычных’ сотрудников***. [[Hamel06]](http://hbr.org/2006/02/the-why-what-and-how-of-management-innovation/ar/1)

> Следовательно, подход *Лин Шесть Сигм (Lean Six Sigma)* рассматривается людьми в Toyota, как просто инструменты [Шести Сигм](https://ru.wikipedia.org/wiki/Шесть_сигм), но не как реальное бережливое мышление. Бывший управляющий завода Toyota и менеджер по персоналу объясняет:

Reviewer: Есть такой устоявшийся термин "Лин Шесть Сигм"?

Author: Как правильно?

> > Lean Six Sigma это подборка инструментов и тренингов, ориентированных на конкретные, изолированные аспекты для снижения стоимости единицы продукции… Подход Toyota [...] гораздо более всеобъемлющий и глубокий.  Отправной точкой является философия Дао Toyota: уважение к людям и непрерывное совершенствование. Этот принцип заключается в выращивании квалифицированных  людей, которые постоянно улучшают процессы. При этом ответственность лежит не на специалистах с "чёрными поясами", но на руководстве, которое запускает весь процесс и является коучами и наставниками. [[LH08]](http://www.amazon.com/Toyota-Culture-Heart-Soul-Way-ebook/dp/B0062Y5VGQ/ref=sr_1_1?ie=UTF8&qid=1414350917&sr=8-1&keywords=The+Heart+and+Soul+of+the+Toyota+Way)

Reviewer: Предлагаю не менять, обсудим.

> > Lean Six Sigma это подборка инструментов и тренингов, ориентированных на конкретные, изолированные аспекты для снижения стоимости единицы продукции… Подход Toyota [...] гораздо более всеобъемлющий и глубокий.  Отправной точкой является философия Дао Toyota: уважение к людям и непрерывное совершенствование. Этот принцип заключается в выращивании квалифицированных  людей, которые постоянно улучшают процессы. При этом ответственность лежит не на специалистах с "чёрными поясами", но на руководстве, которое запускает весь процесс и является коучами и наставниками. [[LH08]](http://www.amazon.com/Toyota-Culture-Heart-Soul-Way-ebook/dp/B0062Y5VGQ/ref=sr_1_1?ie=UTF8&qid=1414350917&sr=8-1&keywords=The+Heart+and+Soul+of+the+Toyota+Way)

Author: ```suggestion
> "Лин Шесть Сигм" это подборка инструментов и тренингов, ориентированных на конкретные, изолированные аспекты для снижения стоимости единицы продукции… Подход Toyota [...] гораздо более всеобъемлющий и глубокий.  Отправной точкой является философия Дао Toyota: уважение к людям и непрерывное совершенствование. Этот принцип заключается в выращивании квалифицированных  людей, которые постоянно улучшают процессы. При этом ответственность лежит не на специалистах с "чёрными поясами", но на руководстве, которое запускает весь процесс и является коучами и наставниками. [[LH08]](http://www.amazon.com/Toyota-Culture-Heart-Soul-Way-ebook/dp/B0062Y5VGQ/ref=sr_1_1?ie=UTF8&qid=1414350917&sr=8-1&keywords=The+Heart+and+Soul+of+the+Toyota+Way)
```

И от генерального директора Toyota Кацуаки Ватанабэ (Katsuaki Watanabe):

> Путь Toyota опирается на два основных столпа: непрерывное совершенствование и уважение к людям. Уважение является обязательным в работе с людьми. Под "людьми" мы подразумеваем сотрудников, партнёров-поставщиков и клиентов. ...Мы не имеем в виду только конечных потребителей; человек на следующем участке конвейера тоже является вашим клиентом. Это способствует командной работе. ***Если вы следуете этому принципу, вы будете также анализировать то, что вы делаете, чтобы понять всё ли вы делаете правильно, таким образом вы не заставите ваших клиентов беспокоиться***. Это подпитывает вашу способность выявлять проблемы и если вы внимательно наблюдаете за вещами, это приведёт к кайдзен — непрерывному совершенствованию. ***Основа Пути Toyota — быть постоянно несогласным со статус-кво; вы постоянно должны задаваться вопросом: “Зачем мы это делаем?”*** (выделение добавлено)

> Уважение к людям и непрерывное совершенствование “ставят под сомнение всё” и создают образ “поддерживающего изменения” мышления. Эти столпы бережливого подхода рассматриваются подробнее далее в этой главе. Если программа внедрения бережливого подхода игнорирует важность этих вещей (мы будем называть такое внедрение бережливого подхода **карго-культом**), тогда существенное понимание и условия для стабильного успеха в бережливом подходе будут упущены.

Reviewer: двое против одного :) вы победили с Алексеем :)

## Почему "Бережливый Подход"? Бережливый Подход и Путь Toyota

Английский термин ‘бережливый’ был придуман для системы Toyota чтобы противопоставить *массовое* производство *немассовому* производству в Toyota — *бережливому* производству. Следствием этого стало радикальное сокращение объёма партии продукции и прекращение попыток конкурировать за счёт экономии на масштабе, а скорее конкурировать за счёт возможности адаптироваться, избегать запасов и работать с очень малыми партиями — вопросы, также поднимаемые в LeSS.

> Двое из авторов книги *The Machine That Changed the World* продолжили писать *"Бережливое Производство"* — популярное вводное издание, в котором резюмируются пять принципов.

Reviewer: ```suggestion
Двое из авторов книги *"Машина, Которая Изменила Мир"* продолжили писать *"Бережливое Производство"* — популярное вводное издание, в котором резюмируются пять принципов.
```
Такая книга переведена на русский, в таком случае я использую русское название: https://www.ozon.ru/context/detail/id/3122505/

> Вообще говоря, глобальной или системной целью бережливого мышления в Toyota является переход “от идеи до прибыли” или “от заказа до прибыли” как можно быстрее и с *устойчивым темпом*, чтобы быстро поставлять ценности (клиенту *и обществу* ) всё больше и больше сокращая время цикла всех процессов, в то же время продолжая обеспечивать высочайшее качество и уровень морального духа. Toyota стремится уменьшать время цикла, но не за счёт "срезания углов" и ухудшения качества, или работая в нестабильном, небезопасном темпе; а скорее, благодаря неустанному *непрерывному совершенствованию*, что требует корпоративной культуры осознанного *уважения к людям*, в которой люди чувствуют, что это безопасно — бросить вызов и изменить статус-кво.

Reviewer: ```suggestion
Вообще говоря, глобальной или системной целью бережливого мышления в Toyota является переход “от идеи к прибыли” или “от заказа к прибыли” как можно быстрее и с *устойчивым темпом*, чтобы быстро поставлять ценности (клиенту *и обществу* ) всё больше и больше сокращая время цикла всех процессов, в то же время продолжая обеспечивать высочайшее качество и уровень морального духа. Toyota стремится уменьшать время цикла, но не за счёт "срезания углов" и ухудшения качества, или работая в нестабильном, небезопасном темпе; а скорее, благодаря неустанному *непрерывному совершенствованию*, что требует корпоративной культуры осознанного *уважения к людям*, в которой люди чувствуют, что это безопасно — бросить вызов и изменить статус-кво.
```

> Всё что мы делаем --- это наблюдаем за промежутком времени, с момента получения заказа от клиента до момента получения денег. И мы уменьшаем этот промежуток времени, сокращая не добавляющие ценности потери.

> Таким образом фокус в бережливом подходе на *эстафетной палочке*, а не на бегунах, устраняя узкие места для ускорения поставки ценности для клиентов, вместо локальной оптимизации, нацеленной на максимизацию загрузки рабочих или машин. Это также является и фокусом в Скраме — поставлять ценный функционал каждую короткую, ограниченную временными рамками, итерацию.

Reviewer: ```suggestion
Таким образом фокус в бережливом подходе на *эстафетной палочке*, а не на бегунах, устраняя узкие места для ускорения поставки ценности для клиентов, вместо локальной оптимизации, нацеленной на максимизацию загрузки рабочих или машин. Это также является и фокусом в Скраме — поставлять ценный функционал каждую короткую, ограниченную временными рамками итерацию.
```
Я вообще сомневаюсь в правильности, но без запятой вроде "звучит" лучше, как думаешь?

Toyota (и её подразделения Lexus и Scion) не только производят автомобили, но также успешно и эффективно *разрабатывают* новые продукты. Принципы бережливого производства также применяются и в продуктовой разработке. Как Toyota достигает “глобальную цель” в двух своих основных процессах: разработке новых продуктов и производстве?

* **Разработка продуктов** — *превзойти конкурентов в обучении*, путём генерации большего количества полезных знаний и их эффективного использования и сохранения.

> * **Производство** — *превзойти конкурентов в улучшении*, фокусируясь на коротких циклах, небольших партиях и очередях, останавливаясь, чтобы найти и исправить корневые проблемы, беспрестанно устраняя все потери (ожидания, лишние передаточные звенья, ...).

Reviewer: ```suggestion
* **Производство** — *превзойти конкурентов в улучшении*, фокусируясь на коротких циклах, небольших партиях и очередях, останавливаясь, чтобы найти и исправить корень проблемы, беспрестанно устраняя все потери (ожидания, лишние передаточные звенья, ...).
```
"корень проблемы" - более точный на мой взгляд перевод.

Исходя из этого, мы пришли к осознанию того, какие управленческие качества необходимы для успешного принятия бережливого подхода, чтобы получить значимый и стабильный успех: команда руководителей не может просто работать в качестве поддержки бережливого подхода в формате «спрашивали-отвечаем». Toyota одна из немногих компаний, которая по всей видимости, демонстрирует эти качества. Подытожим:

* Долгосрочная философия — многие в компании обучаются бережливому мышлению с помощью курсов и менторинга менеджеров-учителей.
* Долгосрочная философия — практически весь менеджмент, включая управляющий уровень, должен иметь чёткое понимание принципов бережливого подхода, жить этим годами и учить этому других.
* Долгосрочная философия — менеджеры-учителя развили навыки системного мышления и улучшения процессов, решения проблем, и они учат этому других. Вся культура пропитана менталитетом и отношением: “Давайте остановимся и поймём корневые причины проблем.”

> **Менеджеры-учителя** и внутренний девиз Toyota: *Правильное Мышление, Хорошие Продукты*. Как они достигают этого “хорошего мышления”, которое лежит в основе их успеха? А достигается это через *культуру наставничества.* Ожидается, что менеджеры должны быть практическими экспертами в своей области работы (как говорится, “мой начальник может выполнить мою работу лучше меня”), ожидается что они должны понимать бережливое мышление и ожидается что они *тратят время на обучение и коучинг остальных*. Во время опросов в Японии мы узнали , что политика управления персоналом в Toyota включает анализ того, сколько времени менеджер тратит времени на обучение. Короче говоря, менеджеры в меньшей степени начальники и в большей степени учителя принципов бережливого мышления, “остановись и исправь сразу,” менталитета *кайдзен*. Таким способом распространяется *ДНК Toyota*.

Reviewer: ```suggestion
**Менеджеры-учителя** и внутренний девиз Toyota: *Правильное Мышление, Хорошие Продукты*. Как они достигают этого “хорошего мышления”, которое лежит в основе их успеха? А достигается это через *культуру наставничества.* Ожидается, что менеджеры должны быть практическими экспертами в своей области работы (как говорится, “мой начальник может выполнить мою работу лучше меня”), ожидается что они должны понимать бережливое мышление и ожидается что они *тратят время на обучение и коучинг остальных*. Во время опросов в Японии мы узнали, что политика управления персоналом в Toyota включает анализ того, сколько времени менеджер тратит времени на обучение. Короче говоря, менеджеры в меньшей степени начальники и в большей степени учителя принципов бережливого мышления, “остановись и исправь сразу,” менталитета *кайдзен*. Таким способом распространяется *ДНК Toyota*.
```
только лишний пробел убрал,  принято )

</td>
<td style="vertical-align:middle;">
<p style="margin-top:0em; padding-top:0em;">

> Этот столп бережливого мышления описан в разделе <a href="./continuous-improvement-towards-perfection.html#RespectforPeople">Уважение к Людям</a> статьи о другом принципе LeSS - Непрерывное Улучшение на Пути к Совершенству.

Reviewer: ```suggestion
Этот столп бережливого мышления описан в разделе <a href="./continuous-improvement-towards-perfection.html#RespectforPeople">Уважение к Людям</a> статьи о другом принципе LeSS &mdash; Непрерывное Улучшение на Пути к Совершенству.
```
А почему бы не длинное тире :) ?

</td>
<td style="vertical-align:middle;">
<p style="margin-top:0em; padding-top:0em;">

> Этот столп бережливого производства изложен в виде отдельного принципа LeSS - <a href="./continuous-improvement-towards-perfection.html">Непрерывное Улучшение на Пути к Совершенству.</a>

Reviewer: ```suggestion
Этот столп бережливого производства изложен в виде отдельного принципа LeSS &mdash; <a href="./continuous-improvement-towards-perfection.html">Непрерывное Улучшение на Пути к Совершенству.</a>
```
длинное тире рулит )

</td>
<td>
<p>

> Обратите внимание, что в первую очередь от лидера ожидается <em>доскональное понимание работы.</em> В Toyota есть такое выражение: "мой начальник может выполнить мою работу лучше меня." В разработке программного обеспечения это означает, что лидер бережливого подхода в совершенстве понимает современные практики разработки. А также лидеры бережливого подхода не делегируют обучение бережливому мышлению "коучам"; <em>они</em> и являются экспертами в бережливом подходе, и они же сами  этому обучают. 

Reviewer: ```suggestion
Обратите внимание, что в первую очередь от лидера ожидается <em>доскональное понимание работы.</em> В Toyota есть такое выражение: "мой начальник может выполнить мою работу лучше меня." В разработке программного обеспечения это означает, что лидер бережливого подхода в совершенстве понимает современные практики разработки. А также лидеры бережливого подхода не делегируют обучение бережливому мышлению "коучам"; <em>они</em> и являются экспертами в бережливом подходе, и они же сами этому обучают. 
```
Принято, просто лишний пробел

</figure>

> **Высокоценная информация** — Несколько идей из бережливого подхода и гибкой разработки помогут разобраться. Например:

Reviewer: ```suggestion
**Бóльшая-ценность информация** — Несколько идей из бережливого подхода и гибкой разработки помогут разобраться. Например:
```
По-моему здесь смысли именно в превосходящем сравнении.

Author: ```suggestion
**Бóльшая ценность информации** — Несколько идей из бережливого подхода и гибкой разработки помогут разобраться. Например:
```

Есть что-то фундаментальное и очень человечное в каденции: Люди ценят или нуждаются в определённых *ритмах* в своей жизни и работе — и ценят или нуждаются в неких *ритуалах* в рамках этих ритмов [[Kerth01]](https://www.amazon.com/Project-Retrospectives-Handbook-Reviews-Paperback/dp/B00ZT1CFE4). Многие из нас работают в состоянии каденции семь дней в неделю. Так, например, существует ритуал еженедельного собрания во вторник утром и другие ему подобные. Проще говоря, каденция в работе улучшает предсказуемость, планирование и взаимодействие. На более глубоком уровне это отражает ритмы нашей жизни.

В больших группах, применяющих LeSS (группы, которые ранее имели небольшую или не тотально-системную каденцию и продолжительные, неограниченные рабочие циклы) зачастую можно услышать от людей: “Обыкновенный короткий Спринт оказался наиболее эффективной вещью, которую мы внедрили.” Это отражает то, насколько высоко люди ценят каденцию.

> Представьте группу, которая не работает в заданных временных рамках, но потенциально может поставлять работающую, протестированную систему в любой час или в любой день (что было бы очень здорово). Представьте, что они хотят проводить координационные встречи по планированию (поскольку вовлечено несколько команд) и они хотят проводить общую ретроспективу. У них есть два варианта: (1) организовывать эти события полуслучайно, или (2) проводить их через регулярные промежутки времени. Этот принцип бережливого подхода рекомендует последний вариант.

Reviewer: ```suggestion
Представьте группу, которая не работает в заданных временных рамках, но потенциально может поставлять работающую, протестированную систему в любой час или в любой день (что было бы очень здорово). Представьте, что они хотят проводить координационные встречи по планированию (поскольку вовлечено несколько команд) и они хотят проводить общую ретроспективу. У них есть два варианта: (1) организовывать эти события наполовину случайно, или (2) проводить их через регулярные промежутки времени. Этот принцип бережливого подхода рекомендует последний вариант.
```

> ##### Каденция и Таймбоксы (Timeboxing)

Reviewer: Есть такой устойчивый термин?

Таймбоксы — это не панацея для всех проблем в работе, но они имеют определённые преимущества:

* Таймбоксы способствуют каденции.

* Работа по разработке ПО — это часто неограниченная (или плохо ограниченная) работа. Когда команда знает, что интервал времени заканчивается 15 марта, это ограничивает нечёткие объёмы работы и увеличивает фокус. Таким образом, временные интервалы ограничивают *расползание объёма работ*, ограничивают "*шлифовку*" (необязательные улучшения, выходящие за границы требований [прим. переводчика]), и увеличивают *фокус*.

* Таймбоксы снижают *аналитический паралич* (чрезмерный анализ или чрезмерное обдумывание, приводящее к "ступору" и затягиванию с принятием решения [прим. переводчика]).

* Представьте вы в университете и у вас назначено задание со сроком до понедельника. Когда вы приступите? Для многих ответом будет: “Ближе к понедельнику.” Это называется *Студенческий Синдром* [[Goldratt97]](https://www.amazon.com/Critical-Chain-Eliyahu-M-Goldratt/dp/0884271536) и таймбоксы являются ему противовесом.

> * Если команды должны поставлять нечто сделанное хорошо точно каждые две недели, потери и неэффективность текущих, обычных способов работы становятся до боли очевидными. Ограничение по времени создаёт движущую силу для улучшений. Это улучшающий эффект “камне в озере”, который рассматривается в [Косвенные Преимущества Уменьшения Объёмов Партии и Продолжительности Цикла](./queueing_theory.html#IndirectBenefitsofReducingBatchSizeandCycleTime).

Reviewer: ```suggestion
* Если команды должны поставлять нечто сделанное хорошо точно каждые две недели, потери и неэффективность текущих, обычных способов работы становятся до боли очевидными. Ограничение по времени создаёт движущую силу для улучшений. Это улучшающий эффект “камни в озере”, который рассматривается в [Косвенные Преимущества Уменьшения Объёмов Партии и Продолжительности Цикла](./queueing_theory.html#IndirectBenefitsofReducingBatchSizeandCycleTime).
```

* *Workplace Management* Тайити Оно (Taiichi Ohno) это небольшая книга от создателя Производственной Системы Toyota (TPS). Она вышла из печати, но недавно была повторно переведена Джоном Миллером (Jon Miller) и сейчас доступна. Книга не много рассказывает от TPS но она содержит серию коротких разделов, которые хорошо демонстрируют, что думает Тайити Оно о системах управления и бережливого производства.

* Книги Мэри и Тома Поппендика *Lean Software Development* и *Бережливое производство программного обеспечения (Implementing Lean Software Development)* — это хорошо написанные книги, которые проводят важную параллель между бережливым производством, системным мышлением и гибкой разработкой.

> Перевод статьи осуществлён [Романом Лапаевым](https://www.linkedin.com/in/romanlapaev) и [Кротовым Артёмом](https://www.facebook.com/artem.v.krotov)

Reviewer: ```suggestion
Перевод статьи осуществлён [Романом Лапаевым](https://www.linkedin.com/in/romanlapaev), [Кротовым Артёмом](https://www.facebook.com/artem.v.krotov) и [Алексеем Ворониным](https://scrumtrek.ru/trainer/4595/aleksey-voronin/)
```
